newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -1,340 +1,340 @@
-          (resourceNames.length > 0
-            ? path.basename(resourceNames[0], path.extname(resourceNames[0]))
-            : objectName),
-        isLooping: direction.isLooping(),
-        externalEditorData,
-      },
-      onChangesSaved: resources => {
-        const newDirection = new gd.Direction();
-        newDirection.setTimeBetweenFrames(direction.getTimeBetweenFrames());
-        newDirection.setLoop(direction.isLooping());
-        resources.forEach(resource => {
-          const sprite = new gd.Sprite();
-          sprite.setImageName(resource.name);
-          // Restore collision masks and points
-          if (
-            resource.originalIndex !== undefined &&
-            resource.originalIndex !== null
-          ) {
-            const originalSprite = direction.getSprite(resource.originalIndex);
-            copySpritePoints(originalSprite, sprite);
-            copySpritePolygons(originalSprite, sprite);
-          } else {
-            if (allDirectionSpritesHaveSamePoints) {
-              copySpritePoints(direction.getSprite(0), sprite);
-            }
-            if (allDirectionSpritesHaveSameCollisionMasks) {
-              copySpritePolygons(direction.getSprite(0), sprite);
-            }
-          }
-          newDirection.addSprite(sprite);
-          sprite.delete();
-        });
-
-        // set metadata if there is such on the direction
-        if (resources[0].metadata) {
-          newDirection.setMetadata(JSON.stringify(resources[0].metadata));
-        }
-
-        // Burst the ResourcesLoader cache to force images to be reloaded (and not cached by the browser).
-        resourcesLoader.burstUrlsCacheForResources(project, resourceNames);
-        onReplaceByDirection(newDirection);
-        // Set optional animation name if the user hasn't done so
-        if (resources[0].newAnimationName) {
-          onChangeName(resources[0].newAnimationName);
-        }
-        newDirection.delete();
-      },
-    });
-  };
-
-  render() {
-    return (
-      <div>
-        <MiniToolbar justifyContent="flex-end" smallest>
-          <DirectionTools
-            direction={this.props.direction}
-            resourcesLoader={this.props.resourcesLoader}
-            project={this.props.project}
-            resourceExternalEditors={this.props.resourceExternalEditors}
-            onEditWith={this.editWith}
-          />
-        </MiniToolbar>
-        <SortableList
-          resourcesLoader={this.props.resourcesLoader}
-          direction={this.props.direction}
-          project={this.props.project}
-          onSortEnd={this.onSortEnd}
-          onAddSprite={this.onAddSprite}
-          selectedSprites={this.props.selectedSprites}
-          onSelectSprite={this.props.onSelectSprite}
-          onSpriteContextMenu={this.props.onSpriteContextMenu}
-          helperClass="sortable-helper"
-          lockAxis="x"
-          axis="x"
-        />
-      </div>
-    );
-  }
-}
+// @flow
```

Comment:
If there is no changes on this file, try to no include it in this PR.

Reply:
I know this is wrong but I don't know what happened